### ISSUE TITLE
json modules

### ISSUE THREAD

**Issue by MylesBorins** [jstime/jstime#85]
There is a proposal -> https://github.com/tc39/proposal-json-modules
It build on top of import assertions -> https://github.com/tc39/proposal-import-assertions

V8 does not yet support import assertions though AFAIK... but I also think that an embedder can choose if they need to require the import assertion or not, need to review the spec.

**Comment by bartlomieju** [comment]
V8 8.9 does support import assertions and should be available in the next release of `rusty_v8` (https://github.com/denoland/rusty_v8/pull/567)

**Comment by MylesBorins** [comment]
OH WOW! That is absolutely incredible, I had no idea V8 moved so quickly on it.

Is it behind a flag? Is the deno team working on implementing standard JSON modules?

**Comment by bartlomieju** [comment]
> Is it behind a flag?

I don't think so, it's [a field on `CreateParams`](https://source.chromium.org/chromium/chromium/src/+/master:v8/include/v8.h;drc=8ae4dc4088826210a52bd678b1921b4e68be2c4b;l=8438). Import assertions are later available on [`ModuleRequest`](https://source.chromium.org/chromium/chromium/src/+/master:v8/include/v8.h;drc=8ae4dc4088826210a52bd678b1921b4e68be2c4b;l=1554) in module resolve callback.

> Is the deno team working on implementing standard JSON modules?

Yep! Once we upgrade `rusty_v8` we will look into implementing that and WASM modules (we were waiting on `v8` to implement import assertions for this purpose 😅 )
